compiler/rustc_target/src/spec/mod.rs

```diff
@@ -1702,6 +1702,7 @@ pub struct TargetOptions {
     pub default_adjusted_cabi: Option<Abi>,
 
     /// Minimum number of bits in #[repr(C)] enum. Defaults to 32.
+    // Is it possible this should default to c_int_width?
```

Comment:
if I understood the code down below correctly, `c_int_width` is the actual default now, right?

Reply:
For targets we take in via a `target-spec.json`, yes.

At the time I kept failing to parse out how to make it happen correctly for builtin targets as well and had initially resolved to fix it later since this only affects 2 builtin targets but if you want I can fix it Properly, I'll just have to take a second look at things, here.

Reply:
I have had a proper look at things.